src/main/java/com/sqltest/demo/model/SalesOrder.java

```diff
@@ -0,0 +1,150 @@
+package com.sqltest.demo.model;
+
+import jakarta.persistence.*;
+
+import java.util.List;
+
+@Entity
```

Comment:
Remove all setter methods and add as annotion.
eg:
@Entity
@Table(name = "eample_table")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Example{

Reply:
All setters are removed and **@Data** annotation is added  to generate getters, setters.